universal-login-react/test/ui/EmailOnboarding.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import {ReactWrapper, mount} from 'enzyme';
+import {MockProvider} from 'ethereum-waffle';
+import {setupSdk} from '@unilogin/sdk/testutils';
+import UniLoginSdk, {WalletService} from '@unilogin/sdk';
+import {EmailFowChooserPage} from '../helpers/pages/EmailFlowChooserPage';
+import {Onboarding} from '../../src';
+
+describe('INT: Email Onboarding', () => {
+  let reactWrapper: ReactWrapper;
+  const createSpy = sinon.spy();
```

Comment:
I would name it `creationSpy` because `create` suggests that something is created in this case spy is created (that is wrong).

Reply:
Or `onCreateSpy`